Makefile

```diff
@@ -91,7 +98,25 @@ $(PDF):
 	ls -l $@
 
 $(FSM): bin
-	PG_AUTOCTL_DEBUG=1 ./src/bin/pg_autoctl/pg_autoctl do fsm gv | dot -Tpng > $@
+	$(PG_AUTOCTL) do fsm gv | dot -Tpng > $@
+
+$(TMUX_SCRIPT): bin
+	mkdir -p $(TMUX_TOP_DIR)
+	$(PG_AUTOCTL) do tmux script \
+         --root $(TMUX_TOP_DIR)  \
+         --nodes $(NODES)        \
+         --layout $(TMUX_LAYOUT) > $@
+
+tmux-script: $(TMUX_SCRIPT) ;
+
+tmux-clean:
+	pkill pg_autoctl || true
+	rm -rf $(TMUX_TOP_DIR)
+
+cluster: install tmux-clean $(TMUX_SCRIPT)
+	mkdir -p $(TMUX_TOP_DIR)/run
+	tmux start-server \; source-file $(TMUX_SCRIPT)
+	pkill pg_autoctl || true
```

Comment:
I found this trick on stackoverflow to send Ctrl+C to all panes on exit:
```
tmux set-hook before-kill-window 'run-shell "
   tmux list-panes -t \"#{window_id}\" -F \"##{pane_id}\" | xargs -I {} tmux send-keys -t {} C-c
"'
```

Source: https://unix.stackexchange.com/a/568928/138818

This would have as the advantage that running `make cluster` doesn't kill pg_autoctl processes from other clusters.

Reply:
When I try to integrate that thing in the tmux script we output, then I get errors that I don't have around anymore. I guess manually killing processes is a better option at the moment.

```
$ cat -n tmux/script-5500.tmux
     2	new-session -s pgautofailover-5500
     3	set-hook before-kill-window 'run-shell "tmux list-panes -t \"#{window_id}\" -F \"##{pane_id}\" | xargs -I {} tmux send-keys -t {} C-c
     4	"'
```